frontend/test/specs/post/timelineDirectiveSpec.js

```diff
@@ -3,24 +3,25 @@
 (describe('Test TimelineController', function() {
     beforeEach(module('app'));
 
-    var timelineCtrl, createController, rootScope;
-    var user = {
+    let timelineCtrl, createController, rootScope, scope, notificationService, postsFactory, q;
+
+    let user = {
```

Comment:
if you are not reassigning any of these objects below, like user, post, etc. use `const` instead of `let`